packages/component-library/src/components/plmg-tabs/plmg-tabs.scss

```
@@ -0,0 +1,86 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Common styles to all shadow-DOM components
+// Remove this section if your component does not use shadow DOM
+// -------------------------------------------------
+:host {
+  display: block;
+  box-sizing: border-box;
+}
+
+:host([hidden]) {
+  display: none;
+}
+// ------------- END Common styles -----------------
+
+// Component style
+// -------------------------------------------------
+.plmg-tabs-list {
+  display: inline-flex;
+  border-bottom: 1px solid tokens.$plmg-color-background-neutral-weak;
```

wrong token I think?
```suggestion
  border-bottom:  border-bottom: 1px solid tokens.$plmg-color-border-neutral-weak;
```

---

Applying the bottom border on the tabs-list creates a single border across the entire width of the tabs. I find this conceptually correct and I don't need to worry about padding or margins if they were to exist on the tab-list. 
Why would you suggest placing it on the tab button?